configs/centerpoint/README.md

```diff
@@ -33,6 +33,70 @@ We follow the below style to name config files. Contributors are advised to foll
 }
 ```
 
+## Usage
+
+### Test time augmentation
+
+We have supported double-flip and scale augmentation during test time. To use test time augmentation by modifying pipeline and test_cfg in the config.
```

Comment:
To use test time augmentation, users need to modify the `test_pipeline` and `test_cfg` in the config.